packages/stores/src/queries-external/ibc/status-by-id.ts

```diff
@@ -42,8 +42,7 @@ class ObservableQueryIbcChainStatus extends ObservableQueryExternalBase<
       );
       if (channelData) {
         if (channelData.size_queue > 15) {
-          if (channelData.duration_minutes > 20) return "blocked";
-          else if (channelData.duration_minutes > 5) return "congested";
+          if (channelData.duration_minutes > 20) return "congested";
```

Comment:
Personally, I'd prefer if we don't change the API and still let it return blocked, instead we just look for contested || blocked (as we do) and update the view accordingly.

Reply:
To be clear, if it returns blocked, you want the button to say congested? Or do you want to button to be updated like 'Withdraw (Channel blocked)' and still say 'blocked'?

Reply:
It can return blocked, but we'll show it as congested in view (as we do)

Reply:
Should we go back to the 5 and 20 mins then? Or bump up to something like 20 and 60 mins?